gno.land/cmd/gnoland/testdata/prevrealm.txtar

```diff
@@ -8,16 +8,14 @@
 # | 2   |          |                     | myrlm.B()       | user address |
 # | 3   |          | through /r/foo      | myrlm.A()       | r/foo        |
 # | 4   |          |                     | myrlm.B()       | r/foo        |
-# | 5   |          | through /p/demo/bar | myrlm.A()       | user address |
+# | 5   | MsgRun   | wallet direct       | myrlm.A()       | user address |
 # | 6   |          |                     | myrlm.B()       | user address |
-# | 7   | MsgRun   | wallet direct       | myrlm.A()       | user address |
-# | 8   |          |                     | myrlm.B()       | user address |
-# | 9   |          | through /r/foo      | myrlm.A()       | r/foo        |
-# | 10  |          |                     | myrlm.B()       | r/foo        |
-# | 11  |          | through /p/demo/bar | myrlm.A()       | user address |
-# | 12  |          |                     | myrlm.B()       | user address |
-# | 13  | MsgCall  | wallet direct       | std.PrevRealm() | user address |
-# | 14  | MsgRun   | wallet direct       | std.PrevRealm() | user address |
+# | 7   |          | through /r/foo      | myrlm.A()       | r/foo        |
+# | 8   |          |                     | myrlm.B()       | r/foo        |
+# | 11  |          | through /p/demo/bar | bar.A()         | user address |
+# | 12  |          |                     | bar.B()         | user address |
+# | 9   | MsgCall  | wallet direct       | std.PrevRealm() | user address |
+# | 10  | MsgRun   | wallet direct       | std.PrevRealm() | user address |
```

Comment:
order and index do not match the actual tests below

Reply:
thanks, fixed